faktorio-fe/src/pages/ContactList/ContactForm.tsx

```diff
@@ -118,9 +128,36 @@ export const ContactForm = ({
           )}
         />
 
+        {displayVatPayer && (
```

Comment:
Thinking of like better API for this "ifing".

```ts
<If condition={}>
  <Then>
  </Then>
  <Else>
  </Else>
</If>
```

Reply:
That's roughly what you could get with

https://www.npmjs.com/package/react-if

but I always thought it would make it even more verbose.
With prettier ternaries and && are usually fairly readable for me.